crates/next-core/src/emit.rs

```diff
@@ -21,13 +21,13 @@ pub fn emit_all_assets(
     node_root: Vc<FileSystemPath>,
     client_relative_path: Vc<FileSystemPath>,
     client_output_path: Vc<FileSystemPath>,
-) -> Vc<Completion> {
-    emit_assets(
+) {
+    let _ = emit_assets(
         all_assets_from_entries(assets),
         node_root,
         client_relative_path,
         client_output_path,
-    )
+    );
```

Comment:
Is this correct? I thought we don't schedule the task for execution until the `Vc` is awaited for the first time?

Reply:
It's scheduled on call. Awaiting only waits for the execution to be completed.